OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDetailFragment.java

```diff
@@ -303,6 +303,10 @@ private void configureEnrollButton() {
 
         if (mEnrolled) {
             mEnrollButton.setText(R.string.view_course_button_text);
+        } else if (courseDetail.invitation_only) {
```

Comment:
Maybe @marcotuts and @gsong can push to review this [PR](https://github.com/edx/edx-platform/pull/15223)